### PR TITLE
Try using vite, seems nice

### PR DESCRIPTION
**Pull request by bjornstar**
Here's step one of converting this to using vite.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (Gusted) [inline index.html:1]: I think the one in `public/index.html` can be removed then if the index.html is in the root folder and was AFAIK specific to CRA behavior.

Comment (bjornstar) [reply index.html:1]: Yeah, I haven't gotten there yet. I just got the code compiling and wanted to put something up.

My favorite part about the PR is the -12k lines in `yarn.lock`

Comment (Gusted) [reply index.html:1]: React-scripts is a pretty heavy one on the dependency side: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json#L30 almost utterly unbelievable.

**Comment by drcmda** [comment]
@bjornstar sorry for breaking it again, had all my locals pushed which fix the race conditions and perf issues that have cropped up

**Comment by bjornstar** [comment]
> @bjornstar sorry for breaking it again, had all my locals pushed which fix the race conditions and perf issues that have cropped up

No problem, got it fixed up now. It's pretty nice, want to give it a try?

**Review by Gusted — APPROVED** [review]
Ding ding. Builds are passing. Transforming the react code seems no problems raises. Just some other issues about too big vendor.js... But that isn't a issues for this PR.

Vite seems to be a great tool for this matter and definitely speeds up the process.

**Comment by drcmda** [comment]
the deployment fails though. it would create a deployment link right in this pr if you fix that one last bit about the build folder.

**Comment by drcmda** [comment]
in the vercel settings i see something that says "override build folder" what is the name of the folder that vite creates?

**Comment by drcmda** [comment]
changed it to "dist" it works! 🎉

**Comment by drcmda** [comment]
hey @bjornstar could you add this? https://vitejs.dev/guide/features.html#jsx

```jsx
// vite.config.js
export default {
  esbuild: {
    jsxInject: `import React from 'react'`
  }
}
```

**Comment by drcmda** [comment]
oh no, i think the names are mangled now

**Comment by drcmda** [comment]
ha, now it runs again. thanks @bjornstar and @Gusted !! vite is super fast! 🎉

**Pull request merged**